frontend/src/scenes/experiments/ExperimentResult.tsx

```diff
@@ -176,6 +176,7 @@ export function ExperimentResult(): JSX.Element {
                             kind: NodeKind.InsightVizNode,
                             source: filtersToQueryNode(transformResultFilters(experimentResults.filters)),
                             showTable: true,
+                            showDateFilter: false,
```

Comment:
I couldn't work out if this prop was even needed. The header is only shown if the query specifies `full` which is not the case here. If the header is not shown then the date filter will never be shown.

@neilkakkar based on https://github.com/PostHog/posthog/pull/7606#discussion_r768620964 - is this the query that was trying to prevent the issue mentioned in the comment? Or should I be looking at a different graph rendered for experiments

Reply:
This is the one!

As long as the experiment still looks like this, with no extra headers: https://storybook.posthog.net/?path=/story/scenes-app-experiments--running-trend-experiment (same for funnels)

We're all good.